backend/tracim_backend/lib/core/workspace.py

```diff
@@ -261,6 +261,32 @@ def execute_created_workspace_actions(self, workspace: Workspace) -> None:
                 except AgendaServerConnectionError as exc:
                     logger.error(self, 'Cannot connect to agenda server')
                     logger.exception(self, exc)
+                except Exception as exc:
+                    logger.error(self, 'Something goes wrong during agenda create/update')
+                    logger.exception(self, exc)
+
+    def execute_update_workspace_actions(self, workspace: Workspace) -> None:
+        from tracim_backend.lib.agenda.agenda import AgendaApi
+        if self._config.CALDAV_ENABLED:
+            agenda_api = AgendaApi(
+                current_user = self._user,
+                session = self._session,
+                config = self._config
+            )
+            if workspace.agenda_enabled:
+                try:
+                    agenda_already_exist = agenda_api.ensure_workspace_agenda_exist(workspace)
+                    if agenda_already_exist:
+                        logger.warning(
+                            self,
+                            'workspace {} is just created but it own agenda already exist !!'.format(workspace.user_id)
+                        )
+                except AgendaServerConnectionError as exc:
+                    logger.error(self, 'Cannot connect to agenda server')
```

Comment:
add url of the agenda server for helping debug

Reply:
@lebouquetin At this level we do not have many information about url. It's also unclear which url you want. The one use by command or the one use by proxy. The second one may be a bit tricky to retrieve here.

Reply:
will fix later #1592